Neos.Flow/Classes/ResourceManagement/EelHelper/StaticResourceHelper.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Flow\ResourceManagement\EelHelper;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Eel\ProtectedContextAwareInterface;
+use Neos\Flow\ResourceManagement\ResourceManager;
+use Neos\Flow\I18n\Service as I18nService;
+
+class StaticResourceHelper implements ProtectedContextAwareInterface
+{
+    /**
+     * @Flow\Inject
+     * @var ResourceManager
+     */
+    protected $resourceManager;
+
+    /**
+     * @Flow\Inject
+     * @var I18nService
+     */
+    protected $i18nService;
+
+    /**
+     * Get the public uri of a package resource
+     *
+     * @param string $packageKey
+     * @param string $pathAndFilename
+     * @param bool $localize
+     * @return string
+     */
+    public function uri(string $packageKey, string $pathAndFilename, bool $localize = false): string
+    {
+        $resourcePath = sprintf('resource://%s/%s', $packageKey, $pathAndFilename);
+        if ($localize === true) {
+            $localizedResourcePathData = $this->i18nService->getLocalizedFilename($resourcePath);
+            $resourcePath = $localizedResourcePathData[0] ?? $resourcePath;
+        }
+        return $this->resourceManager->getPublicPackageResourceUriByPath($resourcePath);
+    }
+
+    /**
+     * Get the content of a package resource
+     *
+     * @param string $packageKey
+     * @param string $pathAndFilename
+     * @param bool $localize
+     * @return string
+     */
+    public function content(string $packageKey, string $pathAndFilename, bool $localize = false): string
+    {
+        $resourcePath = sprintf('resource://%s/%s', $packageKey, $pathAndFilename);
```

Comment:
Please extract the code (similar as in the uri method) for getting the resourcePath to a method, to avoid duplicate :)

Reply:
OK

Reply:
Suppose to include
```
        if ($localize === true) {
            $localizedResourcePathData = $this->i18nService->getLocalizedFilename($resourcePath);
            $resourcePath = $localizedResourcePathData[0] ?? $resourcePath;
    }
```

Reply:
Ahhh sorry, now this is included aswell. Makes totally sense.